tests/helpx/vps.test.js

```diff
@@ -0,0 +1,344 @@
+import { expect, test } from '@playwright/test';
+import { features } from '../../features/helpx/vps.spec.js';
+import vpsPage from '../../selectors/helpx/vps.page.js';
+import helpxconfig from '../../configs/helpx.config.js';
+
+let vps;
+let page;
+let context;
+
+test.beforeAll(async ({ browser }) => {
+  //Verify TOKEN is set at environment variables
+  if (process.env.HLX_TKN !== undefined && process.env.HLX_TKN !== '') {
+    // The environment variable is set and has a non-blank value
+    console.log('Environment variable is set and not blank');
+  } else {
+    // The environment variable is either not set or has a blank value
+    const errorMessage =
+      "Environment variable 'HLX_TKN' is not set or blank. Please ensure it is properly configured.";
+    throw new Error(errorMessage);
+  }
+
+  const authToken = process.env.HLX_TKN;
+  context = await browser.newContext();
+  // Set the authorization token in the header
+  await context.setExtraHTTPHeaders({ authorization: `token ${authToken}` });
+  //create a new page
+  page = await context.newPage();
+  vps = new vpsPage(page); 
+  
+});
+
+test.describe('Verify hidden Tablet and mobile and Desktop for video in Viewport specific container', () => {
+
+  test(`${features[7].name}, ${features[7].tags}`, async ({ baseURL }) => {
+    let vpsTag = features[7].path;
```

Comment:
vpsTag doesn't seem to be changing within your test. 
```suggestion
    const vpsTag = features[7].path;
```